Use `pyvpsolver <https://pypi.python.org/pypi/pyvpsolver/>`_ instead.